### PR TITLE
Fix w2ui menus appearing with unnecessary scrollbars

### PR DESCRIPTION
**Pull request by pedropintosilva**
Before this commit, on chrome/mium, menus such as
image or save as in the tabbed view (and with the browser zoom
set to bellow 80%) were being displayed with scrollbars.

Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: Id32e86229fe5d7aad293d24a20c9bfa555fbe78b

### CODE REVIEW

**Comment by pedropintosilva** [comment]
This needs deep testing, until then we cannot merge it. I did some testing and found no issues with switching the positon but I did not test it as much as I would like

**Review by eszkadev — APPROVED** [review]

**Pull request merged**
